crengine/include/lvrend.h

```diff
@@ -221,7 +219,6 @@ extern int gRenderBlockRenderingFlags;
 // Enable everything
 #define BLOCK_RENDERING_FULL_FEATURED                      0x7FFFFFFF
 
-#define BLOCK_RENDERING_G(f) ( gRenderBlockRenderingFlags & BLOCK_RENDERING_##f )
 #define BLOCK_RENDERING(v, f) ( v & BLOCK_RENDERING_##f )
```

Comment:
May be we could have a few other macros, to reduce the length of the line that do this:
```
BLOCK_RENDERING(getDocument()->getRenderBlockRenderingFlags(), BOX_INLINE_BLOCKS)
BLOCK_RENDERING(enode->getDocument()->getRenderBlockRenderingFlags(), BOX_INLINE_BLOCKS)
could become:
BLOCK_RENDERING_N(this, BOX_INLINE_BLOCKS)
BLOCK_RENDERING_N(enode, BOX_INLINE_BLOCKS)
with 
#define BLOCK_RENDERING_N(n, f) ( n->getDocument()->getRenderBlockRenderingFlags() & BLOCK_RENDERING_##f )

BLOCK_RENDERING(doc->getRenderBlockRenderingFlags(), BOX_INLINE_BLOCKS)) 
could become:
BLOCK_RENDERING_D(this, BOX_INLINE_BLOCKS)
with 
#define BLOCK_RENDERING_D(d, f) ( d->getRenderBlockRenderingFlags() & BLOCK_RENDERING_##f )
```

Reply:
To be honest, I would give them up altogether, they make the code difficult to read.

Reply:
Give up these macros ?! They help readability (a lot!) for me :)